pages/news/hokiport.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Fossil Gen 6 support
+layout: news
+posted: 2023-05-XX
+post: true
+thumbnails: hoki
+summary: The Fossil Gen 6, codenamed "hoki" is now supported by AsteroidOS.
+author: Darrel Griët (MagneFire)
+---
+
+AsteroidOS has been ported to the Fossil Gen 6!
+
+This port is quite unique as it's the first Snapdragon Wear 4100 watch to gain AsteroidOS support.
+
+The port isn't perfect out of the box. Check our [GitHub repository](https://github.com/AsteroidOS/meta-smartwatch/issues?q=is%3Aissue+is%3Aopen+label%3Ahoki) for the current issues..
+
+At the time of writing many of the supported watches aren't available new. This brought up a desire to try and find a newer watch to port. Gaining the knowledge from this could help future porters as well.
+
+The Fossil Gen 6 was specifically chosen in favor of the TicWatch Pro 3 (some of [our community members](https://github.com/AsteroidOS/meta-smartwatch/issues/175) now own this watch) because it exposes a regular USB connection. Making it easier to install AsteroidOS as there's no need for special cables or docks.
+
+# Kernel sources
+
+The challenge began with Fossil Group not having the kernel sources available for download.
+A community member (Zoey) and I contacted Fossil Group separately to inquire for the kernel sources as they legally should because the Linux kernel is licensed under GPL-2.0.
+I didn't think this would be particularly hard since all previous Fossil watches have the kernel sources hosted by [Google](https://android.googlesource.com/kernel/msm/+refs).
+But it took a while for Fossil Group to actually release the kernel sources.
+While Fossil Group mentioned things like: "Fossil supports open source and complies with open source licenses." and "Our Wear OS by Google based smartwatches are using the Linux Kernel which is licensed under GPL." while also linking Google kernel sources they couldn't help directly when it came to the Fossil Gen 6.
+As I contacted Fossil on separate occasions about this I was ensured that this issue was raised internally and I would get an update by e-mail. It took a couple of tries but eventually a case number was created and the conversation continued via e-mail as opposed to their online chat.
+After some back and forth via e-mail kindly asking for the kernel sources they eventually escalated this to the engineering team. This was around the end of 2022. I decided to give Fossil Group some time to get the kernel sources ready. And contacted them again at the beginning of April. At which point they responded to wait a little longer. Two days later the kernel sources were [released](https://github.com/fossil-engineering/kernel-msm-fossil-cw)!
```

Comment:
Maybe we can shorten this story a little bit ? Just explaining that there were a bunch of back and forths and that it took a while without necessarily giving a step by step storyline ?